app/components/public/speaker-item.js

```diff
@@ -5,5 +5,13 @@ export default Component.extend({
   classNames  : ['four wide  speaker column'],
   socialLinks : computed(function() {
     return this.get('speaker').getProperties('twitter', 'facebook', 'github', 'linkedin');
+  }),
+  hasSocialLinks: computed(function() {
+    var currentSpeaker = this.speaker;
+    if (currentSpeaker.twitter || currentSpeaker.facebook || currentSpeaker.github || currentSpeaker.linkedin || currentSpeaker.shortBiography || currentSpeaker.longBiography || currentSpeaker.speakingExperience) {
```

Comment:
@uds5501 , I guess you should just return this expression. Wouldn't it itself return a true or a false.

Reply:
@ritikamotwani Good point. making the changes